### DISCUSSION TITLE
Enable Event Items

### DISCUSSION THREAD

**Discussion by TexasToast03** [YimMenu/HorseMenu#189] [Ideas]
Enable Halloween event for masks, clothing, and horses would be nice. same for the other event items

**Comment by Rxann** [comment]
@TexasToast03 can you make a feature request for this?

**Reply by tuyilmaz** [reply]
I think #173 already includes this.

**Reply by Rxann** [reply]
Okay, never mind then.

**Comment by Rxann** [comment]
Included in #173

**Reply by TexasToast03** [reply]
I dont see anything avout event, just outlaw passes and role xp. Im saying a script to enable halloween masks and clothing. Christmas masks and clothing, and all that other stuff. Like how in gta you can enable halloween event items like the Sanctus which is only available during halloween

**Reply by tuyilmaz** [reply]
Correct me if I'm wrong, but I don't think rdo has special items other than outlaw pass unlocks.

**Reply by Rxann** [reply]
Actually, I think he might be talking about like the satan horns and stuff that people can put on their horses? Those are just masks you buy from madam nazar AFAIK though so.

**Reply by TexasToast03** [reply]
Isnt there also like skull masks or whatever that come out only on halloween?

**Reply by TexasToast03** [reply]
> Actually, I think he might be talking about like the satan horns and stuff that people can put on their horses? Those are just masks you buy from madam nazar AFAIK though so.

And yes, I am

**Reply by tuyilmaz** [reply]
Horse masks are outlaw pass unlocks.


> Isnt there also like skull masks or whatever that come out only on halloween?

Do you mean limited stock items that are enabled in shops for a while?

**Reply by TexasToast03** [reply]
Yes, like event items

**Reply by TexasToast03** [reply]
Things that only come out at a certain time and go away when the time passes

**Reply by tuyilmaz** [reply]
I get it now. I think they're server side, but I'll look into this.

**Reply by Rxann** [reply]
> I get it now. I think they're server side, but I'll look into this.

Thanks. Let me know how it goes. Good luck

**Reply by tuyilmaz** [reply]
After looking at the mp catalog file I've found out that some items are bound to serveronly awards, which means we can't unlock them. However some of them are just items that can be bought like regular clothings. I think if we manage to find a way to buy items using their hashes, we can buy and unlock them too.

**Reply by TexasToast03** [reply]
Damn, well that sucks...
